### DISCUSSION TITLE
New jmk voice for testing

### DISCUSSION THREAD

**Discussion by zstanecic** [RHVoice/RHVoice#616] [Show and tell]
Hi to all!
I am showing you my progress regarding the jmk voice.
This can be installed in form of the NVDA addon for now.
Trained on the original material, dithered and noise shaped.
No additional audio processing used.
This is the JMK voice as referenced in one of the issues here, i don't recall the number.
Download it from the following link:
https://lab.rhvoice.su/s/4p3gKgQXpSeWyRd/download

**Comment by king-dahmanus** [comment]
is this world based? It sounds so muffled even at 100 volume. I think you need to increase the volume range somhow @zstanecic

**Comment by zstanecic** [comment]
Yes, this is world based, on the original samples, recorded to 48 kHz, not on 16 k based, which sound very harsh. That's why i don't want to make the RMS voice out of the existing data, as even on the 24 k sampling rate, it will sound as 16 kHz. Porting of Slt is also in the works.

**Comment by king-dahmanus** [comment]
Ah yes, slt. Nice to know. How about liubov and bdl and the rest? They are 48k based aren't they? Or at least 24k. Anyways, even if the jmk voice is weird as it is, I can still notice the quality improvement that world gives us

**Comment by zstanecic** [comment]
There is also the original bdl available, in the high quality audio. I am also experimenting with it to make it sound more realistic and hq.
In the meantime, please test this new version of jmk, and give the feedback.
https://lab.rhvoice.su/s/eD7KodAmEZDmEEk/download

**Comment by king-dahmanus** [comment]
it doesn't appear to be any different, still sounds a little muffled and low volumed

**Comment by king-dahmanus** [comment]
dear god! It used to read dragon correctly in the other version but now it reads it wrong lol

**Comment by zstanecic** [comment]
This versio was audio processed, nothin more. I will need to investigate the problematic files when it comes to segmenter and htk.
But, these are experimental versions, so, let me normalize the text in the script, and see what can be done actually to fix this, and to label the model correctly.
Actually, we have the situation, where nobody corrected by hand these scripts, or in different words, prompts.

**Comment by king-dahmanus** [comment]
the training thingies you mean?

**Comment by zstanecic** [comment]
exactly.

**Comment by king-dahmanus** [comment]
did you raze an issue to @Olga-Yakovleva  regarding that?

**Comment by zstanecic** [comment]
Hi,
She doesn't have anything with that, as the error can arise in prompts and audio itself, and there are two options:
review the phrases, verify and correct,
or review phrases, and delete them, if these cannot be corrected.

**Comment by king-dahmanus** [comment]
oh you were talking about the data prompts. I misunderstood earlier

**Comment by king-dahmanus** [comment]
how're the voices?

**Comment by zstanecic** [comment]
and, here is the new beta of jmk.
@TheQuinbox 
@king-dahmanus 
@Olga-Yakovleva

**Comment by zstanecic** [comment]
https://lab.rhvoice.su/s/Msexf6Kxq4j37eF/download

**Comment by king-dahmanus** [comment]
sounds good to me, waiting for the other voices to come up. Good job stan

**Comment by trypsynth** [comment]
Ooh, yeah I really like it. Well done! :)
